api/src/main/java/org/openmrs/OrderSet.java

```diff
@@ -96,7 +98,25 @@ public void setOperator(Operator operator) {
 	public void setOrderSetMembers(List<OrderSetMember> orderSetMembers) {
 		this.orderSetMembers = orderSetMembers;
 	}
-	
+
+	/**
+	 * Gets the category
+	 *
+	 * @return the category
+	 */
```

Comment:
Can you add the since annotation? https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-NewPublicMethodsandClasses